### DISCUSSION TITLE
How to upgrade from Laragon 5 to 6

### DISCUSSION THREAD

**Discussion by planetcappe** [leokhoa/laragon#426] [Q&A] [answered]
Hi! I'm using Laragon Full 5.0.0 210523 and I'm wondering if it's worth it upgrading to v6. Also, I'm not sure of where to download and how to install it. Thanks in advance for the help!

**Comment by Pen-y-Fan** [comment, marked as answer]
Laragon 5 was released prior to PHP 8, it has compatibility problems with PHP 8, in particular with Apache and PHP 8.

See [How to update? #405](https://github.com/leokhoa/laragon/discussions/405) for how.
